### PR TITLE
Add the reminder to run VS Perf DDRITs when deployed assemblies change

### LINKED ISSUE #9425: Add a reminder/nag feature to run VS Perf DDRITs when deployed assemblies change
It should be pretty easy to write automation to double-check that we run perf ddrits early when we change the assemblies we ship with (or the .swr in general).

### PR DESCRIPTION
**Pull request by GangWang01**
Fixes #9425

### Context
Add a reminder feature to run VS Perf DDRITs when deployed assemblies change. Double-check that we run perf ddrits early when we change the assemblies we ship with (or the .swr in general).

### Changes Made
Use resourceManagement policy to add a comment to remind the author of pull request that has the change to *.swr file.

### CODE REVIEW

File: .github/policies/resourceManagement.yml [hunk 1]
```diff
@@ -152,5 +152,18 @@ configuration:
...
       description: Comment on vs* branches
+    - if:
+      - payloadType: Pull_Request
+      - or:
+        - isAction:
+            action: Opened
+        - isAction:
+            action: Reopened
+      - filesMatchPattern:
+          pattern: ^.+\.swr$
```
Comment (ladipro) [inline .github/policies/resourceManagement.yml:163]: Technically, any changes to `MSBuild.VSSetup.*` projects could have a similar impact, as could _not_ updating the .swr file when an update is required. Perfectly reasonable as a best-effort reminder, though, thank you!

File: .github/policies/resourceManagement.yml [hunk 2]
```diff
@@ -152,5 +152,18 @@ configuration:
...
+      then:
+      - addReply:
+          reply: Hello @${issueAuthor}, I noticed that you’re changing *.swr file. Please make sure to run VS Perf DDRITs using exp/* insertion before merging the change.
```
Comment (ladipro) [inline .github/policies/resourceManagement.yml:166]: I wonder if a slightly longer comment would make it clearer, especially to folks who are not familiar with our processes:

```suggestion
          reply: Hello @${issueAuthor}, I noticed that you’re changing an *.swr file. Please make sure to validate this change by an experimental VS insertion. This is accomplished by pushing to an exp/* branch, which requires write permissions to this repo.
```

File: .github/policies/resourceManagement.yml (hunk 1 above)
Comment (GangWang01) [reply .github/policies/resourceManagement.yml]: Included this case into the condition.

File: .github/policies/resourceManagement.yml [hunk 3]
```diff
@@ -152,5 +152,21 @@ configuration:
...
+        - isAction:
+            action: Reopened
+      - or:
+        - filesMatchPattern:
+            pattern: ^.+\.swr$
+        - filesMatchPattern:
+            pattern: src/Package/MSBuild.VSSetup.*/.*
+      then:
+      - addReply:
+          reply: Hello @${issueAuthor}, I noticed that you’re changing an *.swr file. Please make sure to validate this change by an experimental VS insertion. This is accomplished by pushing to an exp/* branch, which requires write permissions to this repo.
```
Comment (ladipro) [inline .github/policies/resourceManagement.yml:169]: nit: Maybe replace `*.swr file` with something like `VS setup authoring files` now to reflect the additional pattern?

**Review by ladipro — APPROVED** [review]

**Review by YuliiaKovalova — APPROVED** [review]

**Pull request merged**